lib/core/presentation/widgets/movie_box.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class MovieBox extends StatefulWidget {
```

Comment:
Let's move away from this `Box` naming convention. Try `Card` instead.
Also, this widget doesn't need to be stateful